### PR TITLE
Apply HLint suggestions

### PR DESCRIPTION
**Pull request by Yakshaver-9000**
[Hlint](https://github.com/ndmitchell/hlint) suggested some changes. How do you like them?
If there are some you don't like, you can [dismiss them easily](https://github.com/ArturGajowy/yakshaver#dismissing-select-chagnes) using GitHub UI.
Feel free to close this PR if it's not of help. No feelings will be hurt, promise! :heart:

Yours,
Yakshaver 9000 :red_circle: - *Shaving the yaks, so you don't have to.*

*Have feedback? Just @Yakshaver-9000 in the comments!*

### CODE REVIEW

File: integration-test/PythonSpec.hs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE TupleSections #-}
+
```
Comment (lukasz-golebiewski) [inline integration-test/PythonSpec.hs:1]: Could you remove this newline?

Comment (ArturGajowy) [reply integration-test/PythonSpec.hs:1]: Interesting! This looks like a minor bug in hlint

Comment (ArturGajowy) [inline integration-test/PythonSpec.hs:1]:
```suggestion
```

File: integration-test/IntegrationTest/ThutSpec.hs [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {-# LANGUAGE OverloadedStrings #-}
-{-# LANGUAGE TupleSections     #-}
+
```
Comment (ArturGajowy) [inline integration-test/IntegrationTest/ThutSpec.hs:2]:
```suggestion
```

**Comment by ArturGajowy** [comment]
You should be able to create commits from those suggestions, I apparently lack the permissions :)

**Pull request merged**